src/ctia/bundle/core.clj

```diff
@@ -288,27 +386,153 @@
       (log/warn error)))
   response)
 
+(defn entity->bundle-keys
+  "For given entity key returns corresponding keys that may be present in Bundle schema.
+  e.g. :asset => [:assets :asset_refs]"
+  [entity-key]
+  (let [{:keys [entity plural]} (get (entities/all-entities) entity-key)
+        kw->snake-case-str      (fn [kw] (-> kw name (string/replace #"-" "_")))]
+    [(-> plural kw->snake-case-str keyword)
+     (-> entity kw->snake-case-str (str "_refs") keyword)]))
+
+(s/defn prep-bundle-schema :- s/Any
+  "Remove keys of disabled entities from Bundle schema"
+  [{{:keys [entity-enabled?]} :FeaturesService} :- APIHandlerServices]
+  (->> (entities/all-entities)
+       keys
+       (remove entity-enabled?)
+       (mapcat entity->bundle-keys)
+       (apply st/dissoc NewBundle)))
+
+(s/defschema AssetPropertiesProperties (st/get-in AssetProperties [:properties]))
+
+(s/defn merge-asset_properties-properties :- AssetPropertiesProperties
```

Comment:
that's a very custom interpretation of the model. I expected this merge to happen in IROH

Reply:
Ok, I will add a query param for this behavior. It's too complicated and expensive to do this in IROH.

Reply:
Added:
```
POST /bundle/import?asset_properties-merge-strategy=ignore-existing
- default behavior overwriting all properties
POST /bundle/import?asset_properties-merge-strategy=merge-overriding-previous
- latest property wins
```